nms/app/packages/magmalte/grafana/dashboards/Dashboards.js

```diff
@@ -51,6 +51,22 @@ export const gatewayTemplate: TemplateConfig = variableTemplate({
   includeAll: true,
 });
 
+export const imsiTemplate = variableTemplate({
```

Comment:
Can you just import these templates from CWFDashboards? Either that or delete the code from CWFDashboards and import it there from here

Reply:
@Scott8440 Consolidated the templates here